recipes/newmat/all/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Could you please send this CMakeLists.txt to the author? At least in case is accepted, we no longer need to maintain it.

Reply:
Sure, I will send the final version to the author